todo_app/app.py

```diff
@@ -1,11 +1,38 @@
-from flask import Flask
-
+from flask import Flask, request,render_template
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item, get_item, save_item, delete_item
+
+
 
 app = Flask(__name__)
 app.config.from_object(Config())
+  
 
-
-@app.route('/')
+@app.route('/', methods=['GET','POST'] )
 def index():
-    return 'Hello World!'
+        items = get_items()
+        return render_template ('index.html', my_items = items)
+        
+
+@app.route('/add_tasks', methods=['POST'])
+def add_tasks():
+        add_item(dict(request.form.items())['title'],dict(request.form.items())['priority'], dict(request.form.items())['status'])
```

Comment:
Minor, but there's a lot going on in this line! I think it would be easier to read if we were to split this over a few lines:
```python
        add_item(
                dict(request.form.items())['title'],
                dict(request.form.items())['priority'],
                dict(request.form.items())['status']
        )
```
Taking that a bit further, the `request.form` is a dict (well, technically an [ImmutableMultiDict](https://werkzeug.palletsprojects.com/en/3.0.x/datastructures/#werkzeug.datastructures.ImmutableMultiDict), but we can treat it as a dict!) already, so we can simplify this a bit further:
```suggestion
        add_item(
                request.form['title'],
                request.form['priority'],
                request.form['status']
        )
```